x/mint/types/params.go

```diff
@@ -43,11 +43,11 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:           sdk.DefaultBondDenom,
-		InflationRateChange: sdk.NewDecWithPrec(13, 2),
-		InflationMax:        sdk.NewDecWithPrec(20, 2),
-		InflationMin:        sdk.NewDecWithPrec(7, 2),
+		InflationRateChange: sdk.ZeroDec(),
+		InflationMax:        sdk.NewDecWithPrec(2, 2),
+		InflationMin:        sdk.NewDecWithPrec(2, 2),
```

Comment:
We have not made any decision on the inflation parameters yet. It likely won't be 2% fixed though as that's probably too low.

Reply:
Ok, good to know.

Reply:
this was initially changed because of our [notes](https://github.com/celestiaorg/celestia-app/issues/669#issuecomment-1299045728) indicated heading in this direction (lower the percentage and keep it fixed)

>Decided
we want a lower value for inflation
we want to set inflation max and inflation min to the same value
Undecided
the exact values for inflation are tbd

reverted here https://github.com/celestiaorg/cosmos-sdk/pull/285/commits/ea05762ded38cbf5982da1d796f52463eaa2b0da and we can just rely on changing these params manually per network until we decide on values

Reply:
We do want to reduce the values for inflation - but exact values are tbd as stated in notes

Reply:
My thinking for inflation is a 3-5 % range (not flat). But we will decide this soonish.